crypto/mem.c

```diff
@@ -14,6 +14,17 @@
 #include "internal/cryptlib.h"
 
 /*
+* allow Visual Studio 2003 and higher dump possible memory leaks with _CrtDumpMemoryLeaks() API
+* if OPENSSL_NO_CRYPTO_MDEBUG is defined and library local memory debugging functionality is disabled
+*/
+#if defined(OPENSSL_NO_CRYPTO_MDEBUG) && defined(_DEBUG) && defined(_MSC_VER) && (_MSC_VER >= 1310)
+#include <crtdbg.h>
+#ifdef _NORMAL_BLOCK
+    #define MSVC_MEM_DEBUG
```

Comment:
Please change the indent so that the pound-sign is always in the first column and the word after is indented by one space for each level.  So that this block looks like this:
```
#if defined( ....
# include <crtdbg.h?
# ifdef _NORMAL_BLOCK
#  define MSVC_MEM_DEBUG
# endif
#endif
```

The whole diff here will need that style of change.